lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -160,6 +161,10 @@ function PanelLabel({
   );
 }
 
+type DeepLinkingAPIData = Record<string, unknown> & {
+  auto_grading_config: APIAutoGradingConfig | null;
+};
```

Comment:
For now I have only defined types for  `auto_grading_config`, which is the most complex prop.

We can enhance this type later with the other known props.